lapis/src/main/kotlin/org/genspectrum/lapis/request/CaseInsensitiveFieldsCleaner.kt

```diff
@@ -9,7 +9,11 @@ class CaseInsensitiveFieldsCleaner(
 ) {
     private val fieldsMap = databaseConfig.schema.metadata.map { it.name }.associateBy { it.lowercase() }
 
+    private val phyloTreeFields = databaseConfig.schema.metadata.filter { it.phyloTreeNodeIdentifier }.map { it.name }
```

Comment:
Nitpick: I feel that this field is somewhat misplaced in this class. Why is it here? What does it have to do with the things that are already in this class? It could as well be in the place that uses `getPhyloTreeFields`. (It also doesn't necessarily need a getter for a simple field that could be public)

Reply:
This seemed the neatest way to do this - otherwise we have to pass the databaseConfig everywhere

Reply:
IMO "passing the database config everywhere" doesn't matter. It's Spring bean, it's very cheap to distribute it.

I would rather keep the responsibilities of classes separated properly. We can simply move this method over to the `CaseInsensitiveFieldConverter` and it's done.